drivers/sensor/adltc2990/adltc2990.c

```diff
@@ -253,15 +254,15 @@ static int adltc2990_sample_fetch(const struct device *dev, enum sensor_channel
 		}
 		if (mode_v1_v2 == VOLTAGE_DIFFERENTIAL) {
 			data->pins_v1_v2_values[0] =
-				(adltc2990_fetch_property_value(dev, VOLTAGE_DIFFERENTIAL, V1) *
-				 ADLTC2990_MICROOHM_CONVERSION_FACTOR) /
-				cfg->pins_v1_v2.pins_current_resistor;
+				adltc2990_fetch_property_value(dev, VOLTAGE_DIFFERENTIAL, V1) *
+				(ADLTC2990_MICROOHM_CONVERSION_FACTOR /
+				 cfg->pins_v1_v2.pins_current_resistor);
 		}
 		if (mode_v3_v4 == VOLTAGE_DIFFERENTIAL) {
 			data->pins_v3_v4_values[0] =
-				(adltc2990_fetch_property_value(dev, VOLTAGE_DIFFERENTIAL, V3) *
-				 ADLTC2990_MICROOHM_CONVERSION_FACTOR) /
-				cfg->pins_v3_v4.pins_current_resistor;
+				adltc2990_fetch_property_value(dev, VOLTAGE_DIFFERENTIAL, V3) *
+				(ADLTC2990_MICROOHM_CONVERSION_FACTOR /
+				 cfg->pins_v3_v4.pins_current_resistor);
```

Comment:
Looks like this change should move from the bug fix commit to the refactor commit